src/Mod/Arch/ArchComponent.py

```diff
@@ -1420,11 +1418,7 @@ def claimChildren(self):
         if hasattr(self,"Object"):
             c = []
             if hasattr(self.Object,"Base"):
-                if Draft.getType(self.Object) != "Wall":
-                    c = [self.Object.Base]
-                elif Draft.getType(self.Object.Base) == "Space":
-                    c = []
-                else:
+                if not (Draft.getType(self.Object) == "Wall" and Draft.getType(self.Object.Base) == "Space"):
```

Comment:
for your information, last year @vocx-fc, me and others, possessed by a file splitting demon, we attacked the 7000+ lines modules of Draft wb. As a result of that stilistic change, get_type function was moved to utils module, so you can also call it  like it's done in Draft:
`import draftutils.utils as utils
utils.get_type()`
We didn't touch Arch because @yorikvanhavre already gave this module a more ordered structure.

Reply:
Yeah, I saw the commits. Might be good to change that here as well (OTOH, you could say that `Draft.getType` is a more public API than `draftutils.utils.get_type` and more suited to be called externally), but I'd say this is out of scope for this PR (so I'm resolving this comment). I'll keep it in mind, though.